src/GrumPHP/Task/Codeception.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace GrumPHP\Task;
+
+use GrumPHP\Collection\ProcessArgumentsCollection;
+use GrumPHP\Exception\RuntimeException;
+use GrumPHP\Task\Context\ContextInterface;
+use GrumPHP\Task\Context\GitPreCommitContext;
+use GrumPHP\Task\Context\RunContext;
+
+class Codeception extends AbstractExternalTask
+{
+    const COMMAND_NAME = 'codecept';
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getCommandLocation()
+    {
+        return $this->externalCommandLocator->locate(self::COMMAND_NAME);
+    }
+
+    /**
+     * Default command configuration
+     *
+     * @return array
+     */
+    public function getDefaultConfiguration()
+    {
+        return array(
+            'config_file'   => null,
+            'suite'         => null,
+            'test'          => null,
+            'fail-fast'     => null
+        );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function canRunInContext(ContextInterface $context)
+    {
+        return ($context instanceof GitPreCommitContext || $context instanceof RunContext);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function run(ContextInterface $context)
+    {
+        $files = $context->getFiles()->name('*.php');
+        if (0 === count($files)) {
+            return;
+        }
+
+        // We don't care about changed files here, we want to run the entire suit every time
```

Comment:
A bit of a nit-pick, but we've been removing these comments from the tasks, they're pretty much a copy/paste relic from the past... :)